### DISCUSSION TITLE
DNS가 무엇인가 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#9] [Announcements]
# DNS가 무엇인가 | 정현수 기술 블로그

우리가 이해하기 쉬운 주소로 접속할 수 있는 이유

https://junghyeonsu-dev.vercel.app/posts/What-is-DNS

**Comment by deli-ght** [comment]
와 글 잘읽었습니다 :)
